jenkins/bootstrap_test.py

```diff
@@ -1084,8 +1088,11 @@ def testAllProjectAreUnique(self):
                 if job.startswith('ci-kubernetes-node-'):
                     job = 'ci-kubernetes-node-*'
                 if not line.startswith('#'):
-                    self.assertIn('export', line, line)
-                project = re.search(r'PROJECT="([^"]+)"', line).group(1)
+                  self.assertIn('export', line, line)
+                try:
+                  project = re.search(r'PROJECT="([^"]+)"', line).group(1)
+                except: # catch *all* parsing exceptions
+                  continue
```

Comment:
I added this check to ensure that [check_projects](https://github.com/kubernetes/test-infra/blob/master/jenkins/check_projects.py) works correctly. So I would prefer we do not remove this check.

If you keep it please change the code to not fire a null reference exception. The problem here is the assumption that `re.search()` will always return something.

Also as a general rule code should catch specific exceptions rather than anything (some scenarios require breaking this rule but this is not one of them).

See my comment later in this file for a better way to fix.